vowpalwabbit/gen_cs_example.h

```diff
@@ -11,6 +11,8 @@
 #include "cb_algs.h"
 #include "vw_exception.h"
 
+using namespace std;
```

Comment:
It looks like you are using explicit std::max throughout your code additions. Is the using still necessary? If not, prefer to remove it.